gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/HttpGremlinEndpointHandler.java

```diff
@@ -295,6 +296,9 @@ private GremlinError formErrorResponseMessage(Throwable t, RequestMessage reques
             logger.warn(error.getMessage());
             return error;
         }
+        if (t instanceof GremlinParserException) {
+            return GremlinError.parsing(requestMessage.getGremlin());
```

Comment:
GremlinParserException has a useful message that probably should get included in the response message.

Reply:
I've converted this to simply preserve the original exception message.